doc/man3/SSL_CTX_set_options.pod

```diff
@@ -67,6 +67,12 @@ The following B<bug workaround> options are available:
 Don't prefer ECDHE-ECDSA ciphers when the client appears to be Safari on OS X.
 OS X 10.8..10.8.3 has broken support for ECDHE-ECDSA ciphers.
 
+=item SSL_OP_DISABLE_TLSEXT_CA_NAMES
+
+Disable TLS Extention CA Names. You can want to disable it for security reasons.
+Another usage: some Windows TLS Intplementation crushes if this extention is
+bigger than 1024 bytes.
```

Comment:
Better wording is smth like

Disable TLS Extension CA Names. You may want to disable it for security reasons or for compatibility with some Windows TLS implementations crashing when this extension is larger than 1024 bytes.

Reply:
fixed